Does not support quotes in urls in css

Code like this is not supported:
```css
  @font-face {
    font-family: "MyFont";
    src: url("MyFont.woff");
  }
```

It results in this error:

```
Module not found: Error: Can't resolve './"MyFont.woff"' in 'some/dir/on/machine'
```

Seems like a quick fix in processStyles.

--- a/src/index.js
+++ b/src/index.js
@@ -144,7 +144,7 @@ class DissectHtml {
     const self = this
     // to get -> property: url(filepath)
 
-    const processed = text.replace(/url\(['|']?([^)]+?)['|']?\)/ig, function (_u, url) {
+    const processed = text.replace(/url\(["'|]?([^"'|)]+?)["'|]?\)/ig, function (_u, url) {
       // to get -> filepath from url(filepath), url('filepath') and url('filepath')
       return `url(${self._changeRelUrl(url, path.dirname(cssBasePath))})`
     })

